BE-workspace/src/main/java/com/codesquad/signup4/dto/UserCreateDto.java

```diff
@@ -0,0 +1,89 @@
+package com.codesquad.signup4.dto;
+
+import java.util.List;
+
+public class UserCreateDto {
+    private String userID;
```

Comment:
아래에 isValid를 만드셨는데 api 요청에 받을 dto라면 아래처럼 체크할수도 있어요

위에 관련 문서를 남겨두었습니다. (`@Valid`)

```suggestion
    @NotBlank(message = "userId는 공백일 수 없습니다")
    private String userID;
```